__tests__/getCategories.test.js

```diff
@@ -27,5 +27,20 @@ describe("GET - getCategories", () => {
     })
   });
 
+  describe('GET - endpoints.json', () => {
+    test('should return contents of endpoints.json file ', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            const {endpoints} = response.body
```

Comment:
Good thinking but I would just require it at the top of the test file and assert against that.